EventsExpress/ClientApp/src/containers/inventory-item.js

```diff
@@ -24,9 +24,21 @@ class InventoryItemWrapper extends Component {
         this.onWillTake = this.onWillTake.bind(this);
     }
 
+    shouldComponentUpdate(nextProps, nextState) {
+        if (this.props === nextProps && this.state === nextState)
```

Comment:
You cannot check equality here, since the object memory address will be checked but not the actual values of the props. If you want to check exactly values of the props - you need to do it explicitly for every property of the props. Same for the sate.